ci-operator/step-registry/ipi/openstack/pre/cgroupsv1/ipi-openstack-pre-cgroupsv1-chain.yaml

```diff
@@ -3,6 +3,9 @@ chain:
   steps:
   - chain: ipi-conf-openstack-cgroupsv1
   - chain: ipi-install
+  env:
+  - name: USE_RAMFS
+    default: "true"
```

Comment:
I think these defaults need to be made "false" right?

Reply:
Nope, see https://github.com/openshift/release/pull/42867#issuecomment-1717710974
This goes for all of the below.